jmeter/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Stage image (for downloading files etc)
+FROM openjdk:11.0-jre as stage
```

Comment:
is there a particular reason for not using *jre-slim* here as well, or is this irrelevant for the staging approach used here? Sorry, I am not a Dockerfile expert ...

Reply:
jre-slim had no curl or wget in place. I changed it now to much more lightweight busybox image which should be the right tool for simple download, unzip and copy operations.